app/views/auth_source_ldaps/_form.html.erb

```diff
@@ -24,13 +24,15 @@
                       :data => { :default_ports => AuthSourceLdap::DEFAULT_PORTS } %>
       <%= select_f f, :server_type, AuthSourceLdap::SERVER_TYPES, :first, :last,
                     { :include_blank => _("Choose a server type") },
-                    { :label => _('Server type') } %>
+                    { :label => _('Server type'), :onchange => 'change_ldap_server_type(this)' } %>
     </div>
     <div class="tab-pane" id="account">
       <%= text_f f, :account, :help_inline =>_("Use this account to authenticate, <i>optional</i>").html_safe %>
       <%= password_f f, :account_password, :help_inline => _("Use this account to authenticate, <i>optional</i>").html_safe, :unset => action_name == "edit" %>
       <%= text_f f, :base_dn, :label => _("Base DN"), :size => "col-md-8" %>
       <%= text_f f, :groups_base, :label => _("Groups base DN"), :size => "col-md-8" %>
+      <%= checkbox_f f, :use_netgroups, :help_inline => _("Use NIS netgroups instead of posix groups.") %>
```

Comment:
would you mind adding `:label_help => _('By default we map user groups to standard LDAP Group objects. FreeIPA and POSIX LDAP server types supports alternative way of grouping users through Netgroups. Enable this checkbox if using Netgroups is preferred instead of standard groups.')` (feel free to adjust the text)